src/engine/ruleset/wazuh-core/decoders/core-localfile.yml

```diff
@@ -0,0 +1,62 @@
+name: decoder/core-localfile/0
+# module: wazuh
+
+metadata:
+  title: Localfile queue decoder
+  description: >
+    Process events that has the localfile queue in the standard OSSEC message format.
+    Handles implementation details of Wazuh, not intended for end users.
+  compatibility: Any service being tagged with the queue 49. Wodles and logcollector.
+  versions:
+    - Wazuh 4.*
+  author:
+    name: Wazuh, Inc.
+    date: 03/03/2023
+  references:
+    - https://documentation.wazuh.com/current/development/message-format.html
+
+sources:
+  - decoder/core-ossec-message/0
+
+check:
+  - wazuh.queue: 49
+
+normalize:
+  ### Wodles ###
+  # Command
+  - check:
+      - wazuh.origin: +starts_with/command_
+    map:
+      - wazuh.source: wodle
+      - event.module: $wazuh.origin
+  # Aws
+  - check:
+      - wazuh.origin: +starts_with/aws
+    map:
+      - wazuh.source: wodle
+      - event.module: $wazuh.origin
```

Comment:
Can we merge this into a single check?

Reply:
This logic should further evolve when we start using all sources and wodles